SampleWebApp/App_Start/Startup.Auth.cs

```diff
@@ -37,6 +38,7 @@ public ConditionalMiddlewareInvoker(OwinMiddleware next)
         public void Configuration(IAppBuilder app)
         {
             app.Use(typeof(ConditionalMiddlewareInvoker));
+            app.UseStageMarker(PipelineStage.Authenticate);
```

Comment:
Oh, it's only needed as a proof that it didn't break anything. Not
required. I just couldn't find an easy way to test without creating another
site.

-J

On Wed, Aug 19, 2015 at 2:27 AM, Anders Abel <notifications@github.com>
wrote:

> In SampleWebApp/App_Start/Startup.Auth.cs
> <https://github.com/KentorIT/owin-cookie-saver/pull/5#discussion_r37386269>
> :
>
> > @@ -37,6 +38,7 @@ public ConditionalMiddlewareInvoker(OwinMiddleware next)
> >          public void Configuration(IAppBuilder app)
> >          {
> >              app.Use(typeof(ConditionalMiddlewareInvoker));
> > +            app.UseStageMarker(PipelineStage.Authenticate);
>
> Is the presence of this line this a workaround for the hack with the
> ConditionalMiddlewareInvoker? Is it needed for the proof of concept app to
> work? If not, I'd prefer to have it removed. If it is needed, a comment
> would be appropriate (I'll add a comment on what the
> ConditionalMiddlewareInvoker is too, as it is definitely non-standard).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KentorIT/owin-cookie-saver/pull/5/files#r37386269>.
>